<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>AdjustLogLengthDialog</class>
    <widget class="QDialog" name="AdjustLogLengthDialog">
        <property name="windowTitle">
            <string>Adjust Log Length</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <layout class="QFormLayout" name="formLayout">
                    <item row="0" column="0">
                        <widget class="QLabel" name="logLabel">
                            <property name="text">
                                <string>Log ID / Description:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="1">
                        <widget class="QLabel" name="logInfoLabel">
                            <property name="text">
                                <string>Unknown</string>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="0">
                        <widget class="QLabel" name="lengthLabel">
                            <property name="text">
                                <string>Length to Subtract (in):</string>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="1">
                        <widget class="QDoubleSpinBox" name="lengthSpinBox">
                            <property name="minimum">
                                <double>0.00</double>
                            </property>
                            <property name="maximum">
                                <double>1000.00</double>
                            </property>
                            <property name="singleStep">
                                <double>0.25</double>
                            </property>
                            <property name="value">
                                <double>0.00</double>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <layout class="QHBoxLayout" name="buttonLayout">
                    <item>
                        <spacer name="horizontalSpacer">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QPushButton" name="cancelButton">
                            <property name="text">
                                <string>Cancel</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="okButton">
                            <property name="text">
                                <string>OK</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources />
    <connections />
</ui>